client/my-sites/themes/theme-showcase.scss

```diff
@@ -200,18 +209,23 @@
 		}
 
 		.theme__search-input {
-			flex: 1;
 			width: 100%;
 			padding: 0 0 10px 0;
 			@include break-mobile {
-				flex: 2;
 				padding: 0 10px 0 0;
 			}
 		}
 
 		.section-nav-tabs__dropdown {
-			flex: 1;
 			width: 100%;
+			flex-shrink: 0;
+			@include break-mobile {
+				width: auto;
+			}
+
+			.is-selected {
+				background-color: var(--studio-blue-50);
```

Comment:
I don't think this should be blue, but the same color that uses the categories dropdown:

![image](https://github.com/Automattic/wp-calypso/assets/1233880/f32fa910-d971-427b-a882-a83de05d603e)

Reply:
Also, the categories dropdown has some inner padding which is not present in the pricing dropdown. Ideally both dropdown should be the same.

Reply:
Blue still seems like the most standard color at the moment, we can revisit it later on if we find a styling strategy for our components.